apps/spruce/src/pages/version/VersionTabs.tsx

```diff
@@ -20,20 +20,59 @@ const { parseQueryString } = queryString;
 interface Props {
   taskCount: number;
   isPatch: boolean;
-  // @ts-expect-error: FIXME. This comment was added by an automated script.
-  childPatches: VersionQuery["version"]["patch"]["childPatches"];
+  childPatches: Array<Patch>;
```

Comment:
This isn't actually the type this query is getting. `Patch` represents all of the fields that exist in the patch graphql struct. While `VersionQuery["version"]["patch"]["childPatches"]` represents the fields we are actually querying.

Reply:
You can type ChildPatches like 
```ts
type ChildPatches = NonNullable<
  VersionQuery["version"]["patch"]
>["childPatches"];
```